From: SeongJae Park <sj@kernel.org>
Date: Sun, 27 Oct 2024 14:41:50 -0700
Subject: [PATCH] ==== sample DAMON modules ====

mm/damon: add sample modules

DAMON provides API for other subsystems or modules in the kernel space.
The API functions have their own kernel-doc comments, but having working
example usages would further helps beginners.  There are DAMON modules
built using DAMON API, including DAMON_{SYSFS,RECLAIM,LRU_SORT}.  But
those are for real world usage rather than examples.  Add sample DAMON
modules for exasmple usages of DAMON API for beginners.

Changes from RFC
(https://lore.kernel.org/20241113162624.48973-1-sj@kernel.org)
- Add minimum age to prcl
- Wordsmith commit message
- Remove module_{license,author,description} that unnecessary for static modules

Signed-off-by: SeongJae Park <sj@kernel.org>
---
 damon_meta_changes/xRddXScV | 0
 1 file changed, 0 insertions(+), 0 deletions(-)
 create mode 100644 damon_meta_changes/xRddXScV

diff --git a/damon_meta_changes/xRddXScV b/damon_meta_changes/xRddXScV
new file mode 100644
index 000000000000..e69de29bb2d1
-- 
2.39.5

